### ISSUE TITLE
Feature Request: Permits Animation Queuing

### ISSUE THREAD

**Issue by nitriques** [malsup/cycle#65]
Hi

I use your plugin a lot in contexts sometimes strange (ie. full screen, re-sizable content, ...)

Sometimes, with manual trump active, the animation gets stops and the right effect can't be acheived.

DO you mind if I fork and add this ? Will you merge ?

Thanks.

**Comment by dreadwarrior** [comment]
just found the solution to this probleme here: http://stackoverflow.com/questions/6641954/fancybox-not-playing-nice-with-cycle-plugin-advice

**Comment by nitriques** [comment]
yes, but I would require to be able to specify which value I want for the `queue` param. I already knew how to fix it, I wanted to know if this project's owner wants it...
